rules/library.bzl

```diff
@@ -653,18 +811,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         private_deps.append(swift_angle_bracket_hmap_name)
         _append_headermap_copts(swift_angle_bracket_hmap_name, "-I", additional_objc_copts, additional_swift_copts, additional_cc_copts)
 
-        if module_map:
-            extend_modulemap(
-                name = module_map + ".extended." + name,
-                destination = "%s.extended.modulemap" % name,
-                source = module_map,
-                swift_header = generated_swift_header_name,
-                module_name = module_name,
-                tags = _MANUAL,
-            )
-            module_map = "%s.extended.modulemap" % name
-
-    if cpp_sources and False:
+    if cpp_sources:
```

Comment:
Does your PR need the cc_library? If not, maybe we can enable it in a separate PR to reduce the scope of this PR?

Reply:
Yep will revert this one - I thought this was as a typo on my end originally but it's truly a dead block